python-aexpect.spec

```diff
@@ -1,13 +1,11 @@
-%global srcname aexpect
-
```

Comment:
well the srcname makes sense to me, why removing and hard coding it everywhere?

Reply:
@ldoktor Thanks for reviewing.

I made this change based on a suggestion during the Fedora package review process for related package `python-avocado`: https://bugzilla.redhat.com/show_bug.cgi?id=1930948:

> Opinionated suggestion: The spec file somewhat arbitrarily(?) uses "avacado", "%{srcname}" and "%{pkgname}". I'd stick to "avocado" for better readability, I don't expect the name of the project to be updated so often to justify this complexity.